b2sdk/_test_manager/bucket_tracking.py

```diff
@@ -0,0 +1,32 @@
+######################################################################
+#
+# File: b2sdk/_test_manager/bucket_tracking.py
+#
+# Copyright 2023 Backblaze Inc. All Rights Reserved.
+#
+# License https://www.backblaze.com/using_b2_code.html
+#
+######################################################################
+from b2sdk.v2 import Bucket
+
+
+class BucketTrackingMixin:
+    """
+    Mixin class for B2Api, which enables bucket tracking.
+    This mixin will add a `buckets` member to the B2Api instance and will use it track created and
+    deleted buckets.  The main purpose of this are tests -- the `buckets` member can be used in test
+    teardown to ensure proper bucket cleanup.
```

Comment:
How about adding a statistic of how many buckets in total is created in tests and how many buckets exists at once when tests are ran? This could help us to determine whether we can use `-xdist` to parallelise tests or to run several workflows in parallel.